sdk/nodejs/src/cloudClient.ts

```diff
@@ -61,6 +67,7 @@ export class DVCCloudClient {
                 })
             })
             .catch((err: AxiosError) => {
+                throwIfUserError(err)
```

Comment:
does the axios retry behaviour correctly decide to _not_ try these 4xx errors?

Reply:
our retry condition is 
```
retryCondition: (error) => {
        return !error.response || (error.response.status || 0) >= 500
    }
```
so it shouldn't retry 400 errors